src/uct/sm/scopy/base/scopy_iface.c

```diff
@@ -73,10 +73,11 @@ void uct_scopy_iface_query(uct_scopy_iface_t *iface, uct_iface_attr_t *iface_att
     iface_attr->device_addr_len         = uct_sm_iface_get_device_addr_len();
     iface_attr->ep_addr_len             = 0;
     iface_attr->max_conn_priv           = 0;
-    iface_attr->cap.flags               = UCT_IFACE_FLAG_GET_ZCOPY |
-                                          UCT_IFACE_FLAG_PUT_ZCOPY |
-                                          UCT_IFACE_FLAG_PENDING   |
-                                          UCT_IFACE_FLAG_CONNECT_TO_IFACE;
+    iface_attr->cap.flags               = UCT_IFACE_FLAG_GET_ZCOPY        |
+                                          UCT_IFACE_FLAG_PUT_ZCOPY        |
+                                          UCT_IFACE_FLAG_PENDING          |
+                                          UCT_IFACE_FLAG_CONNECT_TO_IFACE |
+                                          UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE;
```

Comment:
do we really have error reporting from scopy?

Reply:
we expect  that keepalive on sysv/posix would do that

Reply:
but we don't have a guarantee that scopy will be used along with mm?
IMO, scopy should forward err from cma/knem to err callback
or - make PEER_FAILURE flag configurable and off by default